x/superfluid/keeper/grpc_query_test.go

```diff
@@ -329,3 +433,21 @@ func (s *KeeperTestSuite) TestGRPCQueryTotalDelegationByDelegator() {
 		s.Require().True(res.TotalEquivalentStakedAmount.IsEqual(total_osmo_equivalent))
 	}
 }
+
+func arraysMatch(arr1, arr2 []uint64) bool {
```

Comment:
nit: I think there is a helper for that in `osmoutils`:
`ContainsDuplicateDeepEqual`

Reply:
ah nice, was looking for it but did not find it, added here https://github.com/osmosis-labs/osmosis/pull/5600/commits/07074bbf8ac0ac6b3b42712536bb2ea697d89de2